nanoFramework/Amqp.Micro.nanoFramework.nfproj

```diff
@@ -104,38 +104,52 @@
     </Compile>
   </ItemGroup>
   <ItemGroup>
+    <Reference Include="mscorlib">
```

Comment:
here's the reason the app.config: there a duplicate entry for mscorlib

Reply:
Fixed